src/components/sidenav/sidenav.scss

```diff
@@ -107,6 +107,7 @@ $md-sidenav-push-background-color: md-color($md-background, dialog) !default;
     top: 0;
     bottom: 0;
     z-index: 3;
+    min-width: 5%;
```

Comment:
Hm, just questioning -  I don't think that using a percentage is good here. `5%` sounds like a random percentage. 
- Percentage in relative to the parent, do you think that looks / works good on small screens?